schemas/event.yml

```diff
@@ -329,6 +349,13 @@
             Note that pipeline errors that occur during the event ingestion process
             should not use this `event.type` value. Instead, they should use
             `event.kind:pipeline_error`.
+        - name: group
+          description: >
+            The group event type is used for the subset of events within a category
+            that are related to group objects.
+            Common example: `event.category:iam AND event.type:group`.
```

Comment:
Since we're specifically banking on `event.type` being an array here, perhaps we should make the example more explicit in that regard? Perhaps like this: `event.category:iam AND event.type:creation AND event.type:group`

I'd say the same goes for each of the new `event.type`, WDYT?

Reply:
Yes, great idea, will add this.

Reply:
Added array-reinforcing examples for event.type values admin, user, group.